ferrocene/doc/internal-procedures/src/known-problems.rst

```diff
@@ -59,6 +59,14 @@ The `fixed-commit` frontmatter lists the hash of the commit(s) that fixed the
 issue if it has already been fixed.
 Leave it blank if the issue is still open.
 
+frontmatter `duplicate`
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The duplicate field is optional and marks whether a new problem is a duplicate 
+of an older one.
+`introduced-commit` and/or `fixed-commit` may be different; the problem is marked as duplicate if 
+it is marked as duplicate upstream.
```

Comment:
```suggestion
it is considered as duplicate upstream.
```
wording nit, the Rust project has no actual way of "marking" duplicates. They just say duplicate of x and close it (or maybe even leave it open)